### DISCUSSION TITLE
Please update Firefox extension

### DISCUSSION THREAD

**Discussion by bennypowers** [apollographql/apollo-client-devtools#499] [General]
:wave:

The Apollo extension is still at ~2 on the Mozilla add-ons repo, please update it. 

Thank you :)

**Comment by hwillson** [comment]
Thanks @bennypowers - we're held up with a few problems in the Firefox review process, and haven't had a chance to get them resolved yet. They should be fixed this week. 🤞

**Reply by bennypowers** [reply]
I figured something like that. Good luck :)

**Reply by bennypowers** [reply]
@wagnerand PTAL, and the background in https://github.com/apollographql/apollo-client-devtools/issues/73#issuecomment-444497422, if `eval` is the issue

**Comment by jerelmiller** [comment]
Hey all 👋 

I'm doing a bit of cleanup and believe this is no longer an issue as the extension version matches the latest released version. I'm going to go ahead and close this discussion. Thanks for bringing this to our attention!
